### PR TITLE
Upgrade, cleanup, add beatmapsets

### PR DESCRIPTION
**Pull request by ThePooN**
General maintenance and feature requested in #4 (see individual commits).

Test run from staging db: [online.db.zip](https://github.com/ppy/osu-onlinedb-generator/files/9302810/online.db.zip)

### CODE REVIEW

File: Generator.cs [hunk 1]
```diff
@@ -67,34 +68,21 @@ public void Run()
         /// <param name="sqlite"></param>
         private void createSchema(SqliteConnection sqlite)
         {
+            sqlite.Execute(@"CREATE TABLE `osu_beatmapsets` (
+                                  `beatmapset_id` mediumint unsigned NOT NULL,
+                                  `date_submitted` tinyint NOT NULL DEFAULT '0',
```
Comment (peppy) [inline Generator.cs:73]: I'm confused by this column. Why is it `tinyint` and why doesn't it match the naming from the source table?

Comment (ThePooN) [reply Generator.cs]: I used the names you asked for in https://github.com/ppy/osu-onlinedb-generator/issues/4

I have no idea how `tinyint` for a date got in there and how I haven't noticed. Regardless, fixed. Let me know if you want consistent naming with mysql.

Comment (peppy) [reply Generator.cs]: Let's match the naming. I dunno where I got the names I used from, probably from bad memory recollection.

**Comment by ThePooN** [comment]
Sorry for the delay, missed the notification while catching up. Hopefully this is good to go now.

**Review by peppy — APPROVED** [review]

**Pull request merged**